Description: Set maxVersion constraint to 999
Author: Ximin Luo <infinity0@debian.org>
Bug: https://github.com/swick/mozilla-gnome-keyring/issues/43
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/install.rdf
+++ b/install.rdf
@@ -20,7 +20,7 @@
       <Description>
         <em:id>{ec8030f7-c20a-464f-9b0e-13a3a9e97384}</em:id>
         <em:minVersion>4.0</em:minVersion>
-        <em:maxVersion>45.*</em:maxVersion>
+        <em:maxVersion>999.*</em:maxVersion>
       </Description>
     </em:targetApplication>
 
@@ -29,7 +28,7 @@
       <Description>
         <em:id>{3550f703-e582-4d05-9a08-453d09bdfdc6}</em:id>
         <em:minVersion>5.0</em:minVersion>
-        <em:maxVersion>45.*</em:maxVersion>
+        <em:maxVersion>999.*</em:maxVersion>
       </Description>
     </em:targetApplication>
   </Description>
